### PR TITLE
Change env file directory to root home dir

### PR DESCRIPTION
**Pull request by andresbono**

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -e
```
Comment (jotadrilo) [inline run.sh]: let's also check for root access:

```
#!/bin/bash -e

# Make sure only root can run our script
if [[ $EUID -ne 0 ]]; then
   echo "This script must be run as root" 1>&2
   exit 1
fi

# Load ...
```

File: README.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ Your application should include a run.sh script like the one included in this re
 
     ```bash
     # Load deployment environment
-    . /bitnami/.env
+    . /root/.deployment.env
```
Comment (jotadrilo) [inline README.md]: In line 70 we have this:

> The permission of this file are 600 and it is only loaded in the `run.sh` script so this environment variables are not available for users logged in the machine.

What do you think about giving a reasoning? Eg:

> For security reasons, the permissions of this file are 600 and it is only loaded in the `run.sh` script so this environment variables are not available for users logged in the machine.

Note that I also fixed a typo ~permission~ -> permissions

File: run.sh (hunk 1 above)
Comment (andresbono) [reply run.sh]: 👍

File: README.md (hunk 2 above)
Comment (andresbono) [reply README.md]: 👍

**Pull request merged**